apps/src/templates/studioHomepages/SetUpMessage.jsx

```diff
@@ -74,11 +59,11 @@ const SetUpMessage = ({
   onClick,
 }) => (
   <div style={styles.section} className={className}>
-    <div style={styles.wordBox}>
-      <div style={isRtl ? styles.rtlHeading : styles.heading}>
+    <div style={[styles.wordBox, isRtl ? styles.rtl : styles.ltr, {padding:0}]}>
```

Comment:
I wonder if it might be more readable to hoist all these `isRtl` checks outside of the React and put them in variables...